### PR TITLE
Create .thoth.yaml

### PR DESCRIPTION
**Pull request by saisankargochhayat**

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+managers:
```
Comment (fridex) [inline .thoth.yaml:13]: You prepend also configuration for recommendations:

```yaml
host: khemenu.thoth-station.ninja
tls_verify: true
requirements_format: pipenv

runtime_environments:
  - name: ubi8
    operating_system:
      name: ubi
      version: "8"
    python_version: "3.6"
    recommendation_type: latest
```

Comment (fridex) [reply .thoth.yaml:13]: EDIT: updated params

Comment (saisankargochhayat) [reply .thoth.yaml:13]: Done :+1:

Comment (fridex) [reply .thoth.yaml:13]: :+1:

**Review by fridex — APPROVED** [review]
:100:

**Pull request merged**